crypto-ffi/Makefile.toml

```diff
@@ -156,13 +156,23 @@ script = '''
     mv ./bindings/jvm/src/main/kotlin/com/wire/crypto/core_crypto_ffi.kt ./bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCrypto.kt
 
     sed -i.bak \
-        -e 's/\CryptoException\b/CryptoError/g' \
+        -e 's/\bCryptoException\b/CryptoError/g' \
         -e 's/\bE2eIdentityException\b/E2eIdentityError/g' \
         ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt
 
     rm ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt.bak
 '''
 
+[tasks.ffi-kotlin.mac]
+script = '''
+    mv ./bindings/jvm/src/main/kotlin/com/wire/crypto/core_crypto_ffi.kt ./bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCrypto.kt
+
+    sed -i '' \
+        -e 's/[[:<:]]CryptoException[[:>:]]/CryptoError/g' \
+        -e 's/[[:<:]]E2eIdentityException[[:>:]]/E2eIdentityError/g' \
+        ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt
+'''
+
```

Comment:
So we fixing UniFFI not following Swift conventions when generating errors types. Should we also open issue at the UniFFI project?

Reply:
What you mean? Nothing is changing here in behaviors. This is there because MacOS's sed (which is bsd) has a different behavior than Linux's (which is gnu).

Reply:
Ok, then I misunderstood the change.